src/util/isArrayLike.ts

```diff
@@ -0,0 +1 @@
+export const isArrayLike = (<T>(x: any): x is ArrayLike<T> => x && typeof x.length === 'number');
```

Comment:
I'd rather not crowd `from` with barely related defs and end up having split history.